hack/vagrant/README.md

```diff
@@ -0,0 +1,12 @@
+# USAGE
```

Comment:
Could you mind to add more information about Vagrant? For example, the official doc link.

How do you think?